code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -411,6 +275,14 @@
 	force_divisor = 0.5
 	origin_type = /obj/structure/bed/stool/chair
 
+/obj/item/material/stool/chair/fancy
+	name = "fancy chair"
+	desc = "Meeting brawl essential. Now all that's missing is corporate ukulele."
```

Comment:
Flow error. I don't know what this means.

Reply:
It's just the normal chair description modified for humorous purposes.